src/main/scala/intent/sbt/Runner.scala

```diff
@@ -83,12 +83,12 @@ class SbtTask(td: TaskDef, suit: IntentStructure, runner: TestSuiteRunner, focus
   override def taskDef = td
 
   // Tagging tests or suites are currently not supported
-  override def tags(): Array[String] = Array.empty
+  override def tags(): Array[String | Null] = Array.empty
 
-  override def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
+  override def execute(handler: EventHandler | Null, loggers: Array[Logger | Null] | Null): Array[Task | Null] =
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.

**Overridden java method returns array**
Overridden java method returns an array. Java arrays have type `Array[T | Null]`. Returned array must thereby be converted to have type `Array[T | Null]`.